app/views/beta/search_results/show.html.erb

```diff
@@ -1,26 +1,9 @@
 <h1>Search results for '<%= @search_result.search_query_parser_result.corrected_search_query %>'</h1>
 
 <section id="beta-search-results-<%= @search_result.resource_id %>">
-  <div class="govuk-grid-row">
-    <% if @search_result.facet_filter_statistics.any? %>
-      <div class="govuk-grid-column-one-third">
-        <%= render 'sidebar', guide: @search_result.guide %>
-      </div>
-      <div class="govuk-grid-column-two-thirds">
-        <% if @search_result.multiple_headings_view? %>
-          <%= render partial: 'multiple_headings_search_results' %>
-        <% else %>
-          <%= render partial: 'search_results' %>
-        <% end %>
-      </div>
-    <% else %>
-      <div class="govuk-grid-column-full">
-        <% if @search_result.multiple_headings_view? %>
-          <%= render partial: 'multiple_headings_search_results' %>
-        <% else %>
-          <%= render partial: 'search_results' %>
-        <% end %>
-      </div>
-    <% end %>
-  </div>
+  <% if @search_result.hits.commodities.present? %>
+    <%= render 'with_hits' %>
```

Comment:
General comment but if you pass the data into partials rather then relying on globals its easier to reuse partials in other locations in the future - keeps the api surface between view and partial well defined? 

```ruby
<%= render 'with_hits', search_result: @search_result %>
```

Reply:
I'll do this separately but good shout